docs/macros.md

```diff
@@ -959,15 +959,15 @@ ___
 
 ### xts
 
-([view source](https://github.com/Datavault-UK/dbtvault/blob/v0.7.4/macros/tables/xts.sql))
```

Comment:
Please change this. 

We should be using the ([view source]) line. Just change it to 0.7.9. It should just be versioned in advance and expectation of the next release.

Reply:
Uncommented and changed the version to ".../v0.7.9/..."